pac4j-oidc/pom.xml

```diff
@@ -62,6 +69,16 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We try to limit the dependencies pulled by pac4j even for testing. Let's remove this one, as I see it used only for statuses!

Reply:
Also, it is not just the statuses that are being used from the module. `WebServer` extends `NanoHTTPD` which is being used in the test class. So, I think the dependency is required.

Reply:
OK. You're right. In that case, let's put the `nanohttpd.version` property in the main `pom.xml` file...